index.html

```diff
@@ -44,6 +44,217 @@ <h3>I'am a Software Developer</h3>
         </ul>
       </section>
     </section>
+      
+    <section class="container1">
+      <section class="card1">
+        <h2>My Recent Works</h2>
+
+        <hr>
+        
+        <img src="Placeholder.png" alt="">

```

Comment:
- [ ] The width of the `hr` is a lot thicker than what is in the template design.
- [ ] Reduce the distance between the headline and the projects list to match the template design. _See the images below for more clarity._
- [ ] Kindly add an appropriate name for the `alt` attribute. This helps to enhance accessibility and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better. 

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121532829-92da2c00-c9b4-11eb-80fc-beff96764005.png) | ![image](https://user-images.githubusercontent.com/49621279/121532913-a4233880-c9b4-11eb-86c6-aa548f8d8658.png)

Reply:
@chaw-bot No